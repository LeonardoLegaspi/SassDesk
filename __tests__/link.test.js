var sass = require('node-sass');

describe('-----LINK TEST', ()=>{
    test('Links snapshot', async () => {
  
        var result = sass.renderSync({
          data: `
          @import './3_elements/link';
          `,
          outputStyle: 'nested'
        });
        
        expect(result.css.toString()).toMatchSnapshot();
      });
  })