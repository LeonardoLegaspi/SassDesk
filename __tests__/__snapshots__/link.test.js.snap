// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`-----LINK TEST Links snapshot 1`] = `
"/**
 * Calculate the luminance for a color.
 * See https://www.w3.org/TR/WCAG20-TECHS/G17.html#G17-tests
 */
/**
   * Calculate the contrast ratio between two colors.
   * See https://www.w3.org/TR/WCAG20-TECHS/G17.html#G17-tests
   */
/**
   * Determine whether to use dark or light text on top of given color.
   * Returns black for dark text and white for light text.
   */
"
`;
